collections/Laud_Misc/MS_Laud_Misc_626.xml

```diff
@@ -39,20 +42,69 @@
                </msIdentifier>
                <msContents>
                   <msItem n="1" xml:id="MS_Laud_Misc_626-item1">
+                     <locus>(fols. 1r–190v)</locus>
                      <author key="person_69105608">Hugutio of Pisa</author>
                      <title key="work_2175">Liber deriuationum</title>
+                     <msItem>
+                        <locus>(fol. 1r)</locus>
+                        <note>Prologue</note>
+                        <rubric>Incipit prologus super derivationes maiores</rubric>
+                        <incipit> Cum nostri protoplasti suggestiva prevaricatione. <sic>dyabolus</sic> humanum genus a sue dignitatis culmine quam longe <choice><sic>deiecerit</sic> <corr>deciderit</corr></choice> </incipit>
+                        <explicit>a verbo augmenti noftre assertionis auspicium sorciamur</explicit>
+                        <finalRubric>Explicit prologus</finalRubric>
+                        <note>The author is referred to as ‘ … patria Pisanus nomine <sic>Huicio</sic> …’. </note>
```

Comment:
Is it possible to give a reference for this quotation?

Reply:
good idea